src/main/java/frc/robot/Buttons.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj2.command.button.CommandJoystick;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+public class Buttons {
```

Comment:
Perhaps make this more descriptive? Something like `public class ControllerActions`?